client-mvp-04/src/actions/authActions.js

```diff
@@ -0,0 +1,3 @@
+import { LOGIN } from './types';
+
+export const login  =() => ({ type : LOGIN });
```

Comment:
please add an empty line at the end of the file
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file